pom.xml

```diff
@@ -204,15 +204,35 @@
     </scm>
 
     <distributionManagement>
+<!--        <repository>-->
+<!--            <id>bintray-openbankingtoolkit-OpenBankingToolkit</id>-->
+<!--            <name>OpenBankingToolkit UK datamodel</name>-->
+<!--            <url>https://api.bintray.com/maven/openbanking-toolkit/OpenBankingToolKit/openbanking-uk-datamodel/;publish=1</url>-->
+<!--        </repository>-->
         <repository>
-            <id>bintray-openbankingtoolkit-OpenBankingToolkit</id>
```

Comment:
@jgazeleyfr Once this is removed then we won't be deploying to bin tray. Other repositories will not have been yet changed to pull from artifactory, so will not be able to find new versions of things in bintray. To remediate this it would be possible to add the artifactory repo details (see 226 of this file in the new version) to all poms before doing anything else. Then all repos would look in both bintray and artifactory for their versions and it wouldn't matter where they were being published to.